graveler/graveler.go

```diff
@@ -159,10 +159,10 @@ type TagRecord struct {
 
 // Diff represents a change in value based on key
 type Diff struct {
-	Type        DiffType
-	Key         Key
-	Value       *Value
-	OldIdentity []byte // the Identity of the value on the left side of the diff
+	Type         DiffType
+	Key          Key
+	Value        *Value
+	LeftIdentity []byte // the Identity of the value on the left side of the diff
```

Comment:
Why is `Identity` different from a `Value`? why isn't the `Value`field called `LeftValue`? I'm fine with or without left, but I think it should be consistent between Value and Identity...

Reply:
I'll try to explain, maybe you can help me find the right naming.
The "Value" field is typically the value on the right side of the diff (value includes both identity and data) - which is the value "after" the change.
I'm saying typically because in the case of deletion, there is no value on the right side, and in this case you will get the value before the deletion, which is left.

The left identity will give you the identity of the object before the change. It always comes for the left side, and in case of addition (where the left side is empty), you will get a nil array. This field is needed for merge (or any three way diff), where you want to know the identity before the change.